src/pyskindose/main.py

```diff
@@ -3,27 +3,19 @@
 import os
 import pydicom
```

Comment:
reorder imports. See other comment on sections and order

Reply:
done